### PR TITLE
[COMPLETE] More linting fixes

### PR DESCRIPTION
**Pull request by andylolz**
Continuing work from #199…

I’ll just keep adding to this one until I get super bored.

### CODE REVIEW

**Comment by dalepotter** [comment]
Nice work - as a note we tend to use the `Incomplete` label to denote work in progress. Then swap it with `complete` when things are ready for a review :-)

(this is assuming it's possible to add a label when opening a PR as a third party...)

**Comment by andylolz** [comment]
> (this is assuming it's possible to add a label when opening a PR as a third party...)

Unfortunately not – only owners can edit labels.

**Comment by andylolz** [comment]
```
$ flake8 iati/ --count
0
```

🙌

**Comment by andylolz** [comment]
```
$ make pylint
Your code has been rated at 8.14/10 (previous run: 7.58/10, +0.56)
```

Getting closer…

**Comment by dalepotter** [comment]
Thanks for this work @andylolz - it's great to be resolving some of these issues which we've sadly had to de-prioritise for some time...

If you wanted to run the Travis linters on this branch to see how they're reporting after these changes, feel free to add `add-page-models` to the [travis.yml](https://github.com/andylolz/IATI-website/blob/more-linting-fixes/.travis.yml#L52) on lines 52, 58 and 64 as part of this PR.

**Comment by andylolz** [comment]
> If you wanted to run the Travis linters on this branch to see how they're reporting after these changes, feel free to add add-page-models to the travis.yml on lines 52, 58 and 64 as part of this PR.

^^ Ah, that’s a great suggestion!

**Comment by andylolz** [comment]
@allthatilk okay, marking this one as complete. Linting isn’t all passing yet, but getting closer.

File: .pylintrc [hunk 1]
```diff
@@ -20,7 +20,7 @@ persistent=yes
 
 # List of plugins (as comma separated values of python modules names) to load,
 # usually to register additional checkers.
-load-plugins=
+load-plugins=pylint_django
```
Comment (andylolz) [inline .pylintrc]: All of the pylint score improvement is as a result of this.

**Comment by andylolz** [comment]
👍

**Pull request merged**